serviceaccounts.rb

```diff
@@ -9,12 +9,16 @@
 # The test SA key is a special case for local development, and is initialized
 # but not deleted.
 class ServiceAccountContext
+  attr_reader :service_account
 
   SERVICE_ACCOUNT_KEY_PATH = "sa-key.json"
 
   def initialize(project, service_account = nil, path = nil)
     @project = project
     @service_account = service_account
+    if not service_account
+      @service_account = "#{@project}@appspot.gserviceaccount.com"
```

Comment:
Not necessarily for this PR, but could we move knowledge of our actual service account names out of this file so that it could be more portable?

Reply:
I agree this is a bad separation of concerns. I'd want to see a use case and plans for the devops repo before investing on that refactoring.

Reply:
Yeah, it's not a high value activity to make too many changes here right now. 

I think either this is an ad-hoc helper script, in which case it should be kept close to the vest with the deploy code, or it's a generic utility, in which case we should remove assumptions about environments and accounts. I suspect the ops tooling will handle this in a different way, now that I know more about what's involved. Plus, if the token stuff is approved, it may be moot.

Still waiting to hear about my monitoring admin accounts, but my plan was to start fresh for the most part. The trick there is that there's a disinclination to make major changes before launch, so it's a hurry-up-and-wait process.